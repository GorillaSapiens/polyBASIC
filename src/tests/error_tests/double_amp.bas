a = 5.0
b = 4.0
c = a & b
