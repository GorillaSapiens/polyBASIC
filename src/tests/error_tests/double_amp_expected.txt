ERROR: @3:5, UNRECOGNIZED DOUBLE MATH OPERATION ❮&❯
